src/models/db/queries/requests.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../index");
+
+const add = (newRequest) => {
+  return db.oneOrNone(`
```

Comment:
It should be `one`, and not `oneOrNone`, see the API. Because the query always returns one row back, it never returns zero rows.

Reply:
@vitaly-t Thanks for catching that!